data/test-data/test_collect_dict_data.yml

```diff
@@ -0,0 +1,5 @@
+k3: {
```

Comment:
I don't understand this file...some test data for what? Would it make sense to add a comment to the yml file?